src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,29 @@
+package calculator;
+
+import java.util.Scanner;
+
+public class StringCalculator {
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		String inputString = scanner.nextLine();
+
+		System.out.println(calculate(inputString));
+	}
+
+	public static int calculate(String inputString) {
+		String[] components = inputString.split(" ");
+
+		if (components.length < 3) {
+			throw new IllegalArgumentException();
+		}
+
+		Calculator calculator = new Calculator(components[0], components[1], components[2]);
+		int calculateResult = calculator.calculate();
+		for(int i = 3; i < components.length ; i = i + 2) {
+			calculator.setCalculator(calculateResult, components[i], components[i + 1]);
```

Comment:
setter는 사용하지 않습니다
1. 객체에 값을 설정한다
2. 계산한다
이렇게 분리해서 구현하셨는데 그럴 필요는 없어보여요
calculator가 피연산자, 연산자를 인스턴스 변수로 들고 있을 필요도 없어보이네요

```suggestion
			calculateResult = calculator.calculate(calculateResult, components[i], components[i + 1]);
```